src/include/kernel/tokens.h

```diff
@@ -22,6 +22,9 @@
 #define SOF_TPLG_KCTL_ENUM_ID	257
 #define SOF_TPLG_KCTL_BYTES_ID	258
 #define SOF_TPLG_KCTL_SWITCH_ID	259
+#define SOF_TPLG_KCTL_BYTES_VOLATILE_RO 260
+#define SOF_TPLG_KCTL_BYTES_VOLATILE_RW 261
+#define SOF_TPLG_KCTL_BYTES_VOLATILE_WO 262
```

Comment:
Whats the use case for the last one ? I cant think of why we would a control that's both volatile and WO ?

Reply:
My bad that was a copy paste issue, it was supposed to be SOF_TPLG_KCTL_BYTES_WO